package.json

```diff
@@ -29,21 +30,18 @@
     "e2e"
   ],
   "files": [
-    "dist",
```

Comment:
Does probably also need `src/pure.js` and `src/index.js` now

Reply:
True!